app/src/main/java/org/sil/gatherwords/MainActivity.java

```diff
@@ -165,7 +175,85 @@ public void onClick(View v) {
                 }
             });
 
+            ImageButton deleteButton = convertView.findViewById(R.id.session_list_button_delete);
+            deleteButton.setOnClickListener(new View.OnClickListener() {
+                public void onClick(View v) {
+                    Context context = inflater.getContext();
+                    session.deletedAt = new Date();
```

Comment:
Perhaps more of a semantic thing, but I would move this line into `DeleteSessionFromDB.doInBackground()`.  Otherwise, if someone elsewhere tries to execute this task, it would not delete if they forgot to also copy this line.

Reply:
I cannot currently see where something else would call it, but makes logical sense. Updated.